src/js/common/components/RequiredLoginView.jsx

```diff
@@ -57,10 +65,10 @@ class RequiredLoginView extends React.Component {
           attemptingAppealsAccess = requiredApp === 'appeals-status';
         }
 
-        if (this.props.userProfile.status === 'SERVER_ERROR' && !attemptingAppealsAccess) {
+        if (userProfile.status === 'SERVER_ERROR' && !attemptingAppealsAccess) {
           // If va_profile is null, show a system down message.
           return <SystemDownView messageLine1="Sorry, our system is temporarily down while we fix a few things. Please try again later."/>;
-        } else if (this.props.userProfile.status === 'NOT_FOUND' && !attemptingAppealsAccess) {
+        } else if (userProfile.status === 'NOT_FOUND' && !attemptingAppealsAccess) {
```

Comment:
@U-DON could you confirm that this check doesn't interfere with Basic and Advanced users signing in via MHV? Keep in mind these users will NOT have MVI data. I think the comment above is wrong. It says "if va_profile is null, show a system down message." Thats no longer accurate.

If it results in a server error or not found, then that is one thing. But for MHV sign-in users who are not premium, we won't even fetch MVI on the backend so MVI va profile will ALWAYS be null (even though there isn't an error).